apps/frontend/src/libs/components/breadcrumbs/breadcrumbs-item/breadcrumbs-item.tsx

```diff
@@ -0,0 +1,35 @@
+import { type BreadcrumbNavigationItem } from "~/libs/components/breadcrumbs/libs/types/breadcrumb-navigation-item.type.js";
```

Comment:
it's a local type, so use relative path
```suggestion
import { type BreadcrumbNavigationItem } from "../types/types.js";
```